app-text/sgml-common/sgml-common-0.6.3-r6.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit prefix
+
+DESCRIPTION="Base ISO character entities and utilities for SGML"
+HOMEPAGE="http://www.iso.org/standard/16387.html"
+SRC_URI="https://dev.gentoo.org/~floppym/dist/${PN}/${P}-gentoo.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~ppc-aix ~x64-cygwin ~amd64-fbsd ~sparc-fbsd ~x86-fbsd ~amd64-linux ~arm-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~m68k-mint ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+IUSE=""
+
+DEPEND=""
+RDEPEND=""
+
+src_prepare() {
+	# We use a hacked version of install-catalog that supports the ROOT
+	# variable, puts quotes around the CATALOG files, and can be prefixed.
+	cp "${FILESDIR}/${P}-install-catalog.in" "${S}/bin/install-catalog.in" \
+		|| die "Copy of install-catalog.in failed"
+
+	eapply "${FILESDIR}"/${P}-prefix.patch
```

Comment:
ok put this patch in global `PATCHES`, and call `default` here instead. Why you ask? Currently eapply_user fixes will not become `eprefixifed`